src/color.c

```diff
@@ -0,0 +1,99 @@
+/*
+* Copyright 2020 Tresys Technology, LLC
```

Comment:
I am not a lawyer, and could definitely be wrong about this but I think the copyright of the code is owned by the person who wrote it (or their employer if produced as a work for hire), and is not governed on a whole project basis.  Given that you are the sole author of this file, I believe this is your copyright, not Tresys's.  My understanding is that you have the option but not the responsibility to mark your copyright where applicable.  I don't think it's correct to indicate that this file (or color.h) is copyright Tresys though, since the code was not written as a work for hire by a Tresys employee.  It is my understanding that you can leave it blank or mark your own copyright as you see fit.

As I understand it, each contributor (or their employer) owns the copyright for the portions of the code they authored, regardless of marking.  The Copyright notice is just a helpful indicator in that regard.  FWIW here's the Linux Foundation's opinion on the general matter of copyrights: https://www.linuxfoundation.org/blog/2020/01/copyright-notices-in-open-source-software-projects/

ping @stevedlawrence for your thoughts?

Reply:
This is in line with my understanding.

Reply:
Thanks for the advice.
I changed the copyright note to `Copyright 2020 The SELint Contributors`.